lib/route/src/Obelisk/Route.hs

```diff
@@ -312,12 +315,53 @@ isoEncoder f = unsafeMkEncoder $ EncoderImpl
   , _encoderImpl_decode = pure . view (from f)
   }
 
+-- | Lift an 'Iso' over the decoded format to an 'Iso' over encoders
```

Comment:
Can't this be written using existing primitives?  E.g. `isoEncoder` and `(.)`

Even if we want to add this for convenience, it should be implemented that way so that it's obviously correct rather than requiring the reader to understand the details.

Reply:
Oh right. For some reason I thought I couldn't leverage `isoEncoder` here.
The one downside is that now we have stricter constraints we don't seem to actually need.

Reply:
The type of this thing is actually bizarre. It's like it comes from an alternate universe where we somehow use lenses or other optics to manipulate the internals of Encoders directly. You can apply isoEncoder to it and get an Encoder which transforms Encoders to other Encoders. Too higher order! :D